bodhi/server/scripts/approve_testing.py

```diff
@@ -100,14 +121,19 @@ def main(argv=sys.argv):
             # not reached the karma threshold.
             if update.meets_testing_requirements:
                 print(f'{update.alias} now meets testing requirements')
-                text = str(
-                    config.get('testing_approval_msg') % update.mandatory_days_in_testing)
-                update.comment(db, text, author=u'bodhi')
-
-                notifications.publish(
-                    topic='update.requirements_met.stable',
-                    msg=dict(update=update))
-                db.commit()
+                if update.autopush:
+                    update_to_stable(update, db)
+                else:
+                    text = str(
+                        config.get('testing_approval_msg') % update.stable_days
+                    )
```

Comment:
This is an unrelated change in style (and to a style that I personally dislike since it wastes vertical space).